usb/esp_tinyusb/Kconfig

```diff
@@ -160,4 +160,66 @@ menu "TinyUSB Stack"
             help
                 Setting value greater than 0 will enable TinyUSB HID feature.
     endmenu # "HID Device Class (HID)"
+
+    menu "Mass Storage Device Class (MSC)"
+        choice STORAGE_MEDIA
+            prompt "Storage Media Used"
+            default STORAGE_MEDIA_SPIFLASH
+            help
+                Select the storage media that is exposed to USB host.
+
+            config STORAGE_MEDIA_SPIFLASH
+                bool "SPI FLASH"
+
+            config STORAGE_MEDIA_SDMMCCARD
+                bool "SDMMC CARD"
+                depends on IDF_TARGET_ESP32S3
+        endchoice
+
+        if STORAGE_MEDIA_SDMMCCARD
+            choice SDMMC_BUS_WIDTH
```

Comment:
It's okay to use such configuration options in example projects, but for reusable libraries the configuration has to happen at run time, unless there is no technically feasible way of doing it at run time.

We already have an API to initialize the SDMMC host, why not leave this to the caller (i.e. application) and simply accept sdmmc_card_t* as one of the arguments of tinyusb_msc_storage_init_sdcard?

Reply:
Agree.
Now all the configuration of SDCard is done at the Application (example in our case) level.